app/controllers/v1/posts_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::PostsController < ApplicationController
+  before_action :set_user, only: %i[create]
+  load_and_authorize_resource
+
+  def index
+    @posts = User.find(params[:user_id]).posts.includes(:comments)
+    render json: @posts
+  end
+
+  def create
+    @post = @user.posts.build(post_params)
+
+    if @post.save
+      render json: @post, status: :created
+    else
+      render json: { errors: @post.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
+
```

Comment:
- YOu have done great so far :+1: :+1: but there are some requirements you still need to meet before moving to the next milestone. :+1: :100: 

- [ ] Looking at the screenshot below, you can see that I get an error when I try to get posts, same as comments. The error seems to be related to routing `RoutingError (uninitialized constant Api)`.  I'll suggest you review your routes and the folder structure in the controller. :+1: :100: :+1: 

- - Error Screenshot:
![Selection_066](https://github.com/NoelLincoln/blog-app/assets/107632780/18d07e26-7f12-49cd-bba2-9071ab327fc8)